google-cloud-storage/src/test/java/com/google/cloud/storage/it/ITStorageReadChannelTest.java

```diff
@@ -107,6 +108,35 @@ public void storageReadChannel_getObject_returns() throws Exception {
     }
   }
 
+  @Test
+  // @CrossRun.Exclude(transports = Transport.GRPC)
```

Comment:
nit: can we remove this line?

Reply:
Yeah, I'll add it's removal in my next PR (don't want to waste all the cpu time to do a full rebuild and test to remove this comment in a testclass)